qutebrowser/completion/completionwidget.py

```diff
@@ -364,15 +364,13 @@ def set_model(self, model):
         old_model = self.model()
         if old_model is not None and model is not old_model:
             old_model.deleteLater()
-            self._selection_model().deleteLater()
-
-        self.setModel(model)
```

Comment:
Just clarifying ... if we call `oldModel.deleteLater; self.setModel(None)`, you are saying the `deleteLater` never takes effect?

Reply:
The model does still get deleted in that case but I want to avoid the `self.setModel(None)` call because of the whole leaking selection model thing.